clang/test/Analysis/builtin_overflow.c

```diff
@@ -0,0 +1,65 @@
+// RUN: %clang_analyze_cc1 -triple x86_64-unknown-unknown -verify %s \
+// RUN:   -analyzer-checker=core,debug.ExprInspection
+
+#define NULL ((void *)0)
+#define INT_MAX __INT_MAX__
+
+void clang_analyzer_dump_int(int);
+
+void test1(void)
+{
+   int res;
+
```

Comment:
Could you compactify these tests - eg. by putting the curly to the end of the prev line, and dropping the empty newlines?

I'd also suggest the name `test_concrete_add_concrete` or something like that to deliver the intent of the arguments.